tests/Finder/SourceFilesFinderTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright © 2017 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Tests\Finder;
+
+use Infection\Finder\SourceFilesFinder;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @author Lander Vanderstraeten <lander_vanderstraeten@hotmail.com>
+ */
+class SourceFilesFinderTest extends TestCase
+{
+    public function test_it_lists_all_files_without_a_filter()
+    {
+        $sourceFilesFinder = new SourceFilesFinder(['tests/Files/Finder'], ['vendor']);
```

Comment:
AFAICS there is no `vendor` directory in `tests/Files/Finder` so this could be replaced by an empty array (same elsewhere)